portfolio/src/main/webapp/html/comments.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Jay Saleh - Comment</title>
+    <script src="../script/comment-script.js"></script>
+  </head>
+  <body onLoad="getData()">
+    <div id="content">
+      <h1>Fetch Test</h1>
```

Comment:
I am assuming this is temporary? If so, add a TODO to implement properly.

Reply:
done